test/functional/Java8andUp/testng.xml

```diff
@@ -247,14 +247,19 @@
 	</test>
 	<test name="JCL_TEST_Java-Security">
 		<classes>
-			<class name="org.openj9.test.java.security.Test_AccessController"/>
-			<class name="org.openj9.test.java.security.Test_AccessControlContext"/>
 			<class name="org.openj9.test.java.security.Test_MHLoaderConstraints"/>
 			<class name="org.openj9.test.java.security.DigestLoop"/>
 			<class name="org.openj9.test.java.security.OpenSSLInvalidCBCKeySizeTest"/>
 			<class name="org.openj9.test.java.security.OpenSSLInvalidGCMKeySizeTest"/>
 		</classes>
 	</test>
+	<test name="JCL_TEST_Java-Security_SM">
+		<!-- Tests that depend on SecurityManager -->
+		<classes>
+			<class name="org.openj9.test.java.security.Test_AccessController"/>
+			<class name="org.openj9.test.java.security.Test_AccessControlContext"/>
```

Comment:
The security manager related class `AccessController` & `AccessControlContext` will be removed in future release, the test compilation need to be separated as well.

Reply:
I thought we would disable compilation later when the classes are actually removed. (Done in build.xml as a new target=compile section? e.g. `<equals arg1="${JDK_VERSION}" arg2="19" />`

Reply:
https://github.com/eclipse-openj9/openj9/issues/14412#issuecomment-1027506959 predicted the security manager to be removed around JDK19 release.
I think separating the test compilation/running is preferred which is easier for the migration to a Java release w/o security manger classes.

Reply:
Perhaps we could disable the compilation in a separate PR? That way we can disable compilation only when SecurityManager is removed

Reply:
> Perhaps we could disable the compilation in a separate PR?

Yes, disabling the compilation can happen when the classes are removed. 
Separating the compilation of those security manager tests can happen, in one or multiple PRs, which needn't wait until the actual removal.

Reply:
Would I do this by adding a new exclude here with the condition that `${JDK_VERSION}` is >=19? https://github.com/eclipse-openj9/openj9/blob/master/test/functional/Java8andUp/build.xml#L284

Reply:
That works w/ `Test_AccessController` & `Test_AccessControlContext`, there are other not dedicated security manager related tests but contain references to `AccessControlContext`, `AccessController`, `SecurityManager`, etc. such as https://github.com/eclipse-openj9/openj9/blob/99077564bf9bd2a32409b5861e8b8687061cddc0/test/functional/Java8andUp/src_110_up/org/openj9/test/java/lang/Test_Class.java#L710-L711
Those code have to be separated from their existing host classes, it is tedious, not sure if there is a better approach.